cpp/ql/src/Likely Bugs/Memory Management/UsingExpiredStackAddress.ql

```diff
@@ -0,0 +1,245 @@
+/**
+ * @name Use of expired stack-address.
+ * @description Accessing the stack-allocated memory of a function
+ *              after it has returned can lead to memory corruption.
+ * @kind problem
+ * @problem.severity warning
```

Comment:
I'd probably say this is an error?

Reply:
Hm, yes. I think it should be an error, indeed.

Reply:
Fixed in 033edc24f46c2eca7753891564c8557ececd0d84.